lib/sass/calcite-web/components/_dropdown.scss

```diff
@@ -12,59 +12,79 @@
   }
 }
 
+%dot {
```

Comment:
you know, now that I'm looking at it, I think that these classes are global to sass, so we might want to name this something a little more specific as it's pretty likely somebody someday will have a sass class called `dot`. Maybe `active-dot` or `dropdown-dot`. IDK. Maybe @macandcheese will have a good idea on the name

Reply:
Yeah, either of those make sense.

Reply:
%dot is really only used here in one place, it might be useful to define it in a `utils` file to re-use the style in other upcoming updates, like side nav, or filters.

Reply:
ya, that's what I was thinking. @paulcpederson, you good with this?
Also, any thoughts on the class name? is `dot` good?

Reply:
let's do like `active-link-dot` or something more specific

Reply:
done